src/Illuminate/Auth/Access/Gate.php

```diff
@@ -100,20 +100,25 @@ public function has($ability)
      *
      * @param  string  $ability
      * @param  callable|string  $callback
+     * @param  array  $options
      * @return $this
      *
      * @throws \InvalidArgumentException
      */
-    public function define($ability, $callback)
+    public function define($ability, $callback, array $options = [])
     {
+        $options += ['guestable' => false];
+
         if (is_callable($callback)) {
-            $this->abilities[$ability] = $callback;
+            // already callable
```

Comment:
no actually, there's an `else` there which would fall to if we removed this block .. right?